test/data/import/brave-browser-laptop/README

```diff
@@ -0,0 +1,9 @@
+Since Brave browser-laptop is based on Chromium (via Muon/Electron), most of the notes from the ChromeImporter test data README (`../chrome/README`) are also applicable here.
+
+# Generating test data
+
+Unlike Chrome/Chromium, Brave browser-laptop does not support multiple profiles, so if you use Brave as your regular web browser, it may not be immediately obvious how you can create a separate profile for the purpose of generating test data. I found the least obtrusive method to be setting the CHROME_USER_DATA_DIR environment variable, e.g. on macOS:
+
+    TEST_UDD=/path/to/test/user/data/dir
+    mkdir -p $TEST_UDD
+    CHROME_USER_DATA_DIR=$TEST_UDD /Applications/Brave.app/Contents/MacOS/Brave
```

Comment:
Good to know! I didn't know it was also a command line flag, will test again to verify that it works and update these docs.

Reply:
@darkdh As I recall now, I did try to use `--user-data-dir-name` at first, but unfortunately it does not seem to work as intended. It causes Brave to start with a fresh profile, but that profile's data never gets saved in the specified user data directory. After quitting Brave, the specified user data directory is empty. This does not happen when I use `CHROME_USER_DATA_DIR`. Saving the temporary state to disk is critical for my use case, since I need to extract the relevant Brave state files to use for test data.

Reply:
I see, that sounds like a bug in muon

Reply:
@darkdh Upon further testing, it looks like `--user-data-dir` has the expected behavior, while `--user-data-dir-name` does not. `--user-data-dir` is a Chromium [flag](https://peter.sh/experiments/chromium-command-line-switches/#user-data-dir), I'm not sure but it seems `--user-data-dir-name` might have been introduced in Muon. Should I update the documentation to recommend `--user-data-dir` instead of `CHROME_USER_DATA_DIR`?

Reply:
yes, please. thanks